packages/snap/src/config.ts

```diff
@@ -1,5 +1,9 @@
 const config = {
-  dev: true,
+  dev: false,
```

Comment:
We still have config.dev as an (in memory) thing you can access so we dont have to read state every time we check if we are in dev mode (e.g. to check if we should log)

we have a state.devMode also which  this is synchronised with